modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd

```diff
@@ -6,4 +6,5 @@ Export-Package:\
 	${package}.model,\
 	${package}.service,\
 	${package}.service.persistence
--includeresource: META-INF/service.xml=../${artifactId}-service/service.xml
\ No newline at end of file
+-includeresource: META-INF/service.xml=../${artifactId}-service/service.xml
```

Comment:
hey @Camork  can you sort these headers please?  thanks!

Reply:
OK.